packages/test/test-utils/src/timeoutUtils.ts

```diff
@@ -25,9 +25,17 @@ export async function timeoutAwait<T = void>(
     return Promise.race([promise, timeoutPromise<T>(() => { }, timeoutOptions)]);
 }
 
-export async function ensureContainerConnected(container: Container): Promise<void> {
+export async function ensureContainerConnected(
+    container: Container,
+    timeoutOptions: TimeoutWithError = {},
+    failOnContainerClose: boolean = true): Promise<void> {
     if (!container.connected) {
-        return timeoutPromise((resolve) => container.once("connected", () => resolve()));
+        return timeoutPromise((resolve, reject) => {
```

Comment:
Some of the tests that were consolidated use a slightly different check: 

```typescript
if (container.connectionState !== ConnectionState.Connected) {
    ...
}
```

I'm assuming that one and `if (!container.connected)` are equivalent, just leveraging the capabilities of `Container` vs `IContainer`.